Rivet/rivet.ps1

```diff
@@ -159,6 +168,21 @@ if( $PSCmdlet.ParameterSetName -eq 'ShowHelp' )
 
 & (Join-Path -Path $PSScriptRoot -ChildPath Import-Rivet.ps1 -Resolve)
 
+if( $Reset )
+{
+    $confirmation = Read-Host -Prompt "Using the `Reset` switch will drop the database(s) for the current environment. Do you want to proceed? [y/n]"

```

Comment:
Use [`$PSCmdlet.ShouldContinue`](https://docs.microsoft.com/en-us/dotnet/api/system.management.automation.cmdlet.shouldcontinue?view=powershellsdk-7.0.0) instead of `Read-Host`. Implement a `-Force` switch that skips the confirmation (see the docs I linked to).